pkgs/applications/science/math/geogebra/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   name = "geogebra-${version}";
-  version = "5-0-361-0";
+  version = "5-0-369-0";
 
   preferLocalBuild = true;
 
   src = fetchurl {
     url = "http://download.geogebra.org/installers/5.0/GeoGebra-Linux-Portable-${version}.tar.bz2";
-    sha256 = "14kidnk8bidklv474zqipv3f77qnfmc697pl407v0rja481c649a";
+    sha256 = "0b5015z1ff3ksnkmyn2hbfwvhqp1572pdn8llpws32k7w1lb0jnk";
```

Comment:
Can you ask them to not delete old releases. Otherwise I would suggest to mirror the source somewhere else (archive.org).